app/code/Magento/Inventory/README.md

```diff
@@ -1,5 +1,21 @@
-# Inventory
+# Inventory module
 
-**Inventory** provides implementation for inventory management.
-See [concept documentation](https://github.com/magento-engcom/magento2/wiki/Technical-Vision.-Catalog-Inventory)
-for further information.
+The `Inventory` module is part of the MSI (Multi-Source Inventory) project,
```

Comment:
I would re-phrase 
`MSI (Multi-Source Inventory) project` -> New Inventory infrastructure.

It's better to avoid usage of MSI, and start calling new modules as new Inventory